### ISSUE TITLE
"502 bad gateway" error when opening Wordpress admin interface

### ISSUE THREAD

**Issue by eauchat** [YunoHost-Apps/wordpress_ynh#131]
Hey hello,
Thanks for the great package, everything has always been working fairly good so far, but recently a bug came up, that wasn't present before, don't know when it appeared exactly, in a recent update I guess, or maybe I'm doing something wrong but I can't really find what.

I'm using worpdress in "multisite" mode. Wordpress is installed at `mydomain.tld/wp`.
The problem is that, after logging in successfully the `wp-admin/` (or `wp-admin/index.php`) page returns a "502 bad gateway" response.

Two things to specify:
- The problem doesn't occur for the root site: `mydomain.tld/wp/wp-admin/` opens properly, but every other sites admin panel (`mydomain/wp/somesite/wp-admin/`) respond with "502 bad gateway".
- The 502 response seems to occur only when opening `mydomain/wp/somesite/wp-admin/` or `mydomain/wp/somesite/wp-admin/index.php`, I managed to access other parts of the admin panel opening: `mydomain/wp/somesite/wp-admin/edit.php` for example.

The bug occurs both for the wordpress admin user, and with sites admin users.

About wordpress and yunohost. I'm using the last versions, just updated the server. So wordpress 5.7~ynh1. But the problem was also present in 5.6 before upgrading.

If there's anything I'm doing wrong, please tell me. Otherwise I hope the bug report is useful :)

**Comment by kay0u** [comment]
Hi!

I can reproduce the issue, and I found a workaround:

When I try to visit the admin part of a specific site, I have an error message in the file `/var/log/nginx/domain.tld-error.log`
Something like:
> 2021/04/07 13:57:44 [error] 37788#37788: *655 FastCGI sent in stderr: "PHP message: WordPress database error Table 'wordpress.wp_2_auto_updates' doesn't exist for query [...]

My workaround is to disable/re-enable the plugin `Companion Auto Update` in the "Network Admin" area.

I have to redo this dirty hack each time I add a new site.

**Comment by eauchat** [comment]
Yep I confirm that it works for me as well to disable/re-enable the extension. That's cool to have a workaround :)

**Comment by Klodd** [comment]
I think I've encountered a similar problem today, wp-admin pages were failing to load only for children sites (I didn't have a 502 but a "The page isn't redirecting properly")

I fixed it by editing the nginx conf file, to integrate the fix into this repo we should change line 2 of /conf/nginx.conf from :
`	#--MULTISITE--rewrite /wp-admin$ $scheme://$host$uri/ permanent;`
to
`	#--MULTISITE--rewrite /wp-admin$ $scheme://$host$requet_uri/ permanent;`
